src/algebra/lie_algebra.lean

```diff
@@ -378,8 +362,7 @@ An important class of Lie algebras are those arising from the associative algebr
 module endomorphisms.
 -/
 instance of_endomorphism_algebra (M : Type v) [add_comm_group M] [module R M] :
```

Comment:
Do we still need this instance at all? If we're putting a global `has_bracket` instance on all rings, then I think we shouldn't need this any more.

Reply:
Sure I'll remove them